src/Apps/Legion/Components/WizardShell/WizardShell.tsx

```diff
@@ -62,32 +74,43 @@ const WizardShell: React.FC<IWizardShellProps> = (props) => {
         }
     }, [
         wizardNavigationContextState.adapter,
-        wizardNavigationContextState.currentStep
+        wizardNavigationContextState.currentStep,
+        registerNextButtonClick,
+        setIsButtonDisabled
     ]);
 
     logDebugConsole('debug', 'Render');
 
     return (
-        <Stack
-            horizontal={true}
-            tokens={{ childrenGap: 8 }}
-            className={classNames.root}
-        >
-            <div className={classNames.wizardContainer}>
+        <div className={classNames.root}>
+            {/* Left nav */}
+            <div className={classNames.leftNav}>
                 <StepperWizard
                     steps={wizardNavigationContextState.steps}
                     type={StepperWizardType.Vertical}
                     currentStepIndex={wizardNavigationContextState.currentStep}
                 />
             </div>
-            <Separator vertical={true} />
-            {currentPage}
-        </Stack>
+            {/* Top nav */}
+            <div className={classNames.headerNav}>
+                {/* TODO: CREATE DB HEADER */}
+                <Icon iconName={'Globe'} />
+                <h3 className={classNames.headerText}>DATABASE NAME HERE</h3>
```

Comment:
Planning to wrap this in `i18n`?

Reply:
Not this part, since the db name comes from the first step, so it won't be a string